core/src/services/azblob/core.rs

```diff
@@ -369,6 +372,136 @@ impl AzblobCore {
         Ok(req)
     }
 
+    /// Init multipart blob
+    ///
+    /// # Reference
+    ///
+    /// https://learn.microsoft.com/en-us/rest/api/storageservices/put-block-list?
+    pub async fn azblob_init_multipart_blob_request(
```

Comment:
`Put Block List` is similiar to s3 `CompleteUploads`, we can't use it to init multipart blob.